### ISSUE TITLE
Test failures in trace viewer

### ISSUE THREAD

**Issue by Alex319** [catapult-project/catapult#2448]
In ToT trace viewer, if you open the dev server and run the tracing tests, two of the tests will fail:

tracing.ui.timeline_viewport_test.locationObj will fail with 'Cannot read property getContainingTrack of undefined'

tracing.value.ui.scalar_span_test.displaySparkline will fail with 'Expected 0 to equal 1'

**Comment by Alex319** [comment]
I discovered that this was caused by my browser's zoom level not being the default. (But probably this is still a bug because the tests should not depend on zoom level)

**Comment by petrcermak** [comment]
I agree that our tests should support zooming, but I don't have the time to look into it unfortunately. A partial solution would be to at least show a warning infobar at the top of the page.

**Comment by benshayden** [comment]
Maybe the test harness could set document.body.style.zoom to the ratio between a hard-coded table of default devicePixelRatios and the current window.devicePixelRatio when !tr.isHeadless? E.g. the default devicePixelRatio on the (ahem) Pixel 2 is 2, so set document.body.style.zoom = 2 / window.devicePixelRatio;
